src/dashboards/components/AutoRefreshOverlay.tsx

```diff
@@ -97,6 +110,39 @@ const AutoRefreshOverlay: FC = () => {
     'Hours'
   )
 
+  const refreshOptions = [
+    {
+      id: 'indefinite-auto-refresh',
+      title: 'Indefinite',
+      active: infiniteDuration,
```

Comment:
nit: since the infiniteDuration solely reflects the tab state, can we rename this to be reflective of the tab state? Not blocking for this work, just a suggestion